### PR TITLE
Remove msg-105

### PR DESCRIPTION
**Pull request by wurzka**
see #111

### CODE REVIEW

**Review by albertmink — APPROVED** [review]
Looks good to me.

File: src/zaff_tools.msag.xml [hunk 1]
```diff
@@ -20,12 +20,6 @@
...
     </T100>
-    <T100>
-     <SPRSL>E</SPRSL>
-     <ARBGB>ZAFF_TOOLS</ARBGB>
-     <MSGNR>105</MSGNR>
-     <TEXT>Component &amp;1 of constant &amp;2 in ABAP Doc link doesn&apos;t exist</TEXT>
-    </T100>
     <T100>
      <SPRSL>E</SPRSL>
      <ARBGB>ZAFF_TOOLS</ARBGB>
```
Comment (Singh212dab) [inline src/zaff_tools.msag.xml:25]: </T100>

**Review by Singh212dab — CHANGES_REQUESTED** [review]

File: src/zaff_tools.msag.xml (hunk 1 above)
Comment (wurzka) [reply src/zaff_tools.msag.xml:25]: I'm sorry, what change do you suggest?

Comment (Singh212dab) [reply src/zaff_tools.msag.xml:25]: </T100> at line 26 to close it.maybe

**Review by Singh212dab — CHANGES_REQUESTED** [review]

File: src/zaff_tools.msag.xml (hunk 1 above)
Comment (wurzka) [reply src/zaff_tools.msag.xml:25]: In my opinion, this message class is correct, so I would resolve this conservation und merge this pull request. Feel free to raise an issue if you see any further problem.

**Pull request merged**